src/components/basic/DlPanelContainer/DlPanelContainer.vue

```diff
@@ -268,6 +276,51 @@ export default defineComponent({
             return {
                 [this.position]: '0px'
             }
+        },
+        innerContainerStyles(): Record<string, string> {
+            return {
+                textAlign: 'left',
+                fontSize: 'var(--dl-font-size-body)',
+                position: 'relative',
+                padding: '0px 10px',
+                height: '100%',
+                boxSizing: 'border-box',
+                color: 'var(--dl-color-darker)',
+                backgroundColor: 'var(--dl-color-panel-background)',
+                zIndex: 'var(--dl-z-index-panel)',
+                width: this.isStudioLeftPanel
+                    ? 'calc(var(--dl-panel-container-width) - 10px)'
+                    : this.width + 'px'

```

Comment:
The `width` was not defined, will it affect other places ?

Reply:
it wont have any affect since it has default value .